README-dev.md

```diff
@@ -1,98 +1,136 @@
 # Mina
 
-Mina is a new cryptocurrency protocol with a lightweight, constant-sized blockchain.
+Mina is a cryptocurrency protocol with a lightweight, constant-sized blockchain.
 
-- [Node Developers Overview Page](https://docs.minaprotocol.com/node-developers)
-- [Repository Readme](README.md)
+- [Node Developers Overview](https://docs.minaprotocol.com/node-developers)
+- [Mina README](README)
```

Comment:
I shared a video with you in the #mf-o1-docs channel on Slack

Reply:
As I mentioned on Slack, in my opinion the link should not only work in web browsers, but also in off-line editors that developers use to edit code (VS Code, Emacs and Vim seem to be the most popular at east among Mina engineers). Both web browsers and editors vary wildly in functionality, therefore the fact it works for you does not necessarily mean it'll work for everyone. Thus, it's best to stick to the filename exactly so that it works for the maximum number of setups people might use. Unfortunate as it is, this is the reality.